playwright/tests/organize/campaigns-list/speed-dial.spec.ts

```diff
@@ -40,12 +40,17 @@ test.describe('All campaigns page speed dial', () => {
         'input:near(#visibility)',
         ReferendumSignatures.visibility
       );
-      await page.click('button > :text("Submit")');
+
+      await Promise.all([
```

Comment:
The `waitForNavigation()` need to be _not_ concurrent in this case, so that's why I had to do it this way. Two concurrent "threads", one of them with serial `waitForNavigation()`s in  it.